open_prices/api/proofs/views.py

```diff
@@ -126,8 +126,8 @@ def upload(self, request: Request) -> Response:
     def process_with_gemini(self, request: Request) -> Response:
         files = request.FILES.getlist("files")
         sample_files = [PIL.Image.open(file.file) for file in files]
-        res = extract_from_price_tags(sample_files)
-        return Response(res, status=status.HTTP_200_OK)
+        labels = [extract_from_price_tag(sample_file) for sample_file in sample_files]
+        return Response({"labels": labels}, status=status.HTTP_200_OK)
```

Comment:
the response changes ? so that we change the frontend integration

Reply:
The response is exactly the same, no? The `Labels` TypedDict doesn't exist anymore

Reply:
nevermind, no changes :)